src/machine_helpers.py

```diff
@@ -28,3 +35,26 @@ def update_mongod_service(
     if role == Config.Role.CONFIG_SERVER:
         mongos_start_args = get_mongos_args(config, snap_install=True)
         add_args_to_env("MONGOS_ARGS", mongos_start_args)
+
+
+def setup_logrotate_and_cron() -> None:
+    """Create and write the logrotate config file."""
+    logger.debug("Creating logrotate config file")
+
+    with open("templates/logrotate.j2", "r") as file:
+        template = jinja2.Template(file.read())
+
+    rendered = template.render(
+        logs_directory=f"{MONGODB_COMMON_DIR}/{LOG_DIR}",
+        mongo_user=MONGO_USER,
+    )
+
+    with open("/etc/logrotate.d/mongodb", "w") as file:
+        file.write(rendered)
```

Comment:
noob question. What is this? Can you add comments for it 🙏 

Specifically answering these questions:
1. what is the significance of `templates/logrotate.j2`, what is its orgin? 
2. what happens during rendering

Reply:
During rendering the template variables will be replaced with actual values as you can see.

`logrotate.j2`  is a template for configuration for logrotate based on different docs about log rotation for mongodb.

This approach (rendering template) and updating the cron job is the same as in MySQL